CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @nikhalim
+* @CHrislian-NI

```

Comment:
If you want to add `CHrislian-NI` to help with your review load, it has to be specified on the same line. Maybe this is what you intended?

From the docs: 

```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2
```

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners